tic_tac_toe/game.py

```diff
@@ -9,7 +23,12 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
+    result = None
+    if board[position[0]][position[1]] == '-':
+        result = True
+    else:
+        result = False
+    return result
```

Comment:
It would be cleaner to just use `return board[position[0]][position[1]] == '-'` to the same ends